examples/files/basic_golang_app/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/nexB/go-inspector/basicGoApp
```

Comment:
Are these examples or the start of a tests suite?

Reply:
This is an example of an application, which must be compiled and after it is compiled with the help of GO Inspector you can see which libraries were used in this application.